molecule/securedrop-docker/create.yml

```diff
@@ -0,0 +1,57 @@
+---
+- name: Create
+  hosts: localhost
+  connection: local
+  gather_facts: False
+  no_log: "{{ not lookup('env', 'MOLECULE_DEBUG') | bool }}"
+  vars:
+    molecule_file: "{{ lookup('env', 'MOLECULE_FILE') }}"
+    molecule_ephemeral_directory: "{{ lookup('env', 'MOLECULE_EPHEMERAL_DIRECTORY') }}"
+    molecule_scenario_directory: "{{ lookup('env', 'MOLECULE_SCENARIO_DIRECTORY') }}"
+    molecule_yml: "{{ lookup('file', molecule_file) | molecule_from_yaml }}"
+  pre_tasks:
+    - name: Inspect kernel configuration for presence of Retpoline setting
```

Comment:
Thanks for moving this to `create` :+1:

Reply:
Great strategy, solid approach. We could also mandate use a grsecurity-kernel here, since that's been a security control for us to date, but that's a separate matter.